05_deploy_bootstrap_vm.sh

```diff
@@ -115,7 +115,7 @@ for i in 0 1 2; do
   NODE_IP="172.22.0.2${i}"
   HOSTNAME="${CLUSTER_NAME}-etcd-${i}.${BASE_DOMAIN}"
   # Make sure internal dnsmasq would assign an expected IP
-  echo "dhcp-host=${NODE_MAC},${HOSTNAME},${NODE_IP}" >> /tmp/dnsmasq.conf
+  echo "dhcp-host=${NODE_MAC},${HOSTNAME},${NODE_IP}" | sudo tee /tmp/dnsmasq.conf
```

Comment:
```suggestion
  echo "dhcp-host=${NODE_MAC},${HOSTNAME},${NODE_IP}" | sudo tee -a /tmp/dnsmasq.conf
```

Without the `-a` you are not preserving the script semantics

Reply:
Would it be better to just remove the sudo from the cp, since this file doesn't really need to be owned by root while in /tmp?

But otherwise yeah agree we need -a to append in this loop